### DISCUSSION TITLE
Unable to Access the Video Manual

### DISCUSSION THREAD

**Discussion by yuma116** [HumanSupportRobot/QandA#93] [Q&A]
I am attempting to view the video manual available on hsr.io, but I am unable to reach the location where the video is hosted on ownCloud. Are there others encountering this same problem? I would greatly appreciate any assistance.

**Comment by yoshimi-iyoda** [comment]
Please let me check.
Can you still not access the site after following the steps below?


![image](https://github.com/HumanSupportRobot/QandA/assets/71624812/c9b8d3ad-0282-4b20-a2a0-654f0ad838a3)


If you still cannot access the site, could you please post the error screen?
